python/cudf_polars/cudf_polars/callback.py

```diff
@@ -104,8 +104,8 @@ def default_memory_resource(
         ):
             raise ComputeError(
                 "GPU engine requested, but incorrect cudf-polars package installed. "
-                "If your system has a CUDA 11 driver, please uninstall `cudf-polars-cu12` "
-                "and install `cudf-polars-cu11`"
+                "If your system has a CUDA `x` driver, please uninstall `cudf-polars-cu12` "
```

Comment:
How can this error be reached? Do we need this to say “Requires CUDA 12+” instead of recommending a reinstall?

I tried to trace down how this code is reached. Is it one of these two paths? https://github.com/search?q=repo%3Arapidsai%2Frmm+%22not+supported+with+this+CUDA+driver%2Fruntime+version%22&type=code

Reply:
Yes, that is the paths. I updated the error message.